src/lib/client.js

```diff
@@ -95,12 +95,15 @@ export default class Client {
       .catch(err => reject(err)));
   }
 
-  enhanced_pdp_modal(charges) {
+  enhanced_pdp_modal(charges, type = 'pdp') {
     if (charges && typeof charges !== 'object') {
       return reject('charges should be a charges object');
     }
 
-    const url = pdpHost(ui, this.platform) + '/pdp/' + this.key + '/' + [ charges.data.total, charges.data.shipping, charges.data.tax, charges.data.grand_total ].join(',');
+    allowedTypes = ['pdp', 'cart'];
+    if (!allowedTypes.includes(type)) return false;
```

Comment:
is this also supposed to be carried over to the built files?  should it `reject` like the other checks, or does returning `false` work?